api/bus.go

```diff
@@ -144,6 +145,22 @@ type UpdateBlocklistRequest struct {
 	Remove []string `json:"remove"`
 }
 
+// AccountsUpdateBalanceRequest is the request type for /accounts/:id/add
+// endpoint.
+type AccountsUpdateBalanceRequest struct {
+	Host   types.PublicKey `json:"host"`
+	Owner  string          `json:"owner"`
+	Amount *big.Int        `json:"amount"`
```

Comment:
maybe we should add a custom api param type for this like we did with duration and time?

Reply:
Any particular reason? `big.Int` already implements the needed interfaces and represents the number as a string.

Reply:
in that case it's fine